documentation/examples/prometheus-kubernetes.yml

```diff
@@ -1,22 +1,61 @@
-# A scrape configuration for running prometheus in cluster on kubernetes. It
-# will create endpoints for node and master roles, as well as any service
-# which is annotated with `prometheus_io_scrape=true`
+# A scrape configuration for running prometheus in cluster on kubernetes.
```

Comment:
Hmm, maybe "in cluster on Kubernetes" -> "on a Kubernetes cluster" or "in the cluster on Kubernetes"?

Capitalize proper nouns Prometheus and Kubernetes please.